### PR TITLE
Add REST API events

### PR DESCRIPTION
**Pull request by dnarain-astro**
Adding REST API events.  

@chomipi88  Need some feedback on whether an event should have both its v1 and v2 fields included since a customer could have both in their logs - e.g delete user has migrated from v1 API to v2 API. Should we have some sort of policy on how long it remains as a v1 and v2 API event (audit logs are 90 days old, so conceivably, we should remove it from the v1 API list 90 days after it was added to v2)?

### CODE REVIEW

File: astro/audit-logs.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ The following categories of event data are available in the audit log:
 - Airflow UI access: The data generated when users access the Airflow UI.
 - Astronomer container registry access: The data generated when users access the Astronomer container registry with the Astro CLI.
 
-The audit log file is provided as a JSON file. Every entry in the audit log file corresponds to an event, and event attributes provide additional information about the specific event. 
+The audit log file is provided as a newline limited JSON (NDJSON)file. Every entry in the audit log file corresponds to an event, and event attributes provide additional information about the specific event.
```
Comment (kwcanuck) [inline astro/audit-logs.md:22]:
```suggestion
The audit log file is provided as a newline delimited JSON (NDJSON) file. Every entry in the audit log file corresponds to an event, and event attributes provide additional information about each event. 
```

File: astro/audit-logs.md [hunk 2]
```diff
@@ -37,7 +37,11 @@ The following table lists the common fields shared by all three categories of ev
 
 #### API event fields
 
-The following table lists the fields that are unique to API events.
+Audit Log events may be generated from either the v1 API or the v2 API.   Each API generates different fields for the same actions and your audit log may include a collection of events from either API.
```
Comment (kwcanuck) [inline astro/audit-logs.md:40]:
```suggestion
Audit log events can be generated from the v1 API or the v2 API. Each API generates different fields for the same actions, and your audit log might include events from both APIs.  
```

File: astro/audit-logs.md [hunk 3]
```diff
@@ -37,7 +37,11 @@ The following table lists the common fields shared by all three categories of ev
 
 #### API event fields
 
-The following table lists the fields that are unique to API events.
+Audit Log events may be generated from either the v1 API or the v2 API.   Each API generates different fields for the same actions and your audit log may include a collection of events from either API.  
+
+This following is a list of all API events; your audit log may contain a mix of both v1 and v2 API events.
```
Comment (kwcanuck) [inline astro/audit-logs.md:42]:
```suggestion

```

File: astro/audit-logs.md [hunk 4]
```diff
@@ -68,6 +72,27 @@ The following table maps some common `operationName` attributes to their corresp
 | An API key is deleted for a Deployment. | `deleteDeploymentApiKey` | `id` |
 | The code for a Deployment is updated. | `ImageCreate` | `deploymentId` |
 
+##### v2 API event fields 
+
+The following table lists the fields that are unique to v1 API events.
```
Comment (kwcanuck) [inline astro/audit-logs.md:77]:
```suggestion
The following table lists the fields that are unique to v2 API events.
```

File: astro/audit-logs.md [hunk 5]
```diff
@@ -68,6 +72,27 @@ The following table maps some common `operationName` attributes to their corresp
...
+
+| Field  | Description                                          |
+| ------- | ---------------------------------------------------- |
+| `path` | The path to the REST API that is invoked. |
```
Comment (kwcanuck) [inline astro/audit-logs.md:81]:
```suggestion
| `path` | The path to the invoked REST API. |
```

**Review by kwcanuck — APPROVED** [review]

File: astro/audit-logs.md (hunk 2 above)
Comment (chomipi88) [reply astro/audit-logs.md:39]: Let's make it clear that we are transitioning from v1 and v2. Over time, we expect to output less and less v1 API events.

**Review by chomipi88 — COMMENTED** [review]
lgtm

**Pull request merged**